pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -2215,67 +2222,18 @@ spec:
 	var wantEvents []string
 	reconciledRun, _ := prt.reconcileRun("foo", "test-pipeline-run-pending", wantEvents, false)
 
-	checkPipelineRunConditionStatusAndReason(t, reconciledRun, corev1.ConditionUnknown, v1beta1.PipelineRunReasonPending.String())
+	checkPipelineRunConditionStatusAndReason(t, reconciledRun, corev1.ConditionUnknown, v1.PipelineRunReasonPending.String())
 
 	if reconciledRun.Status.StartTime != nil {
 		t.Errorf("Start time should be nil, not: %s", reconciledRun.Status.StartTime)
 	}
 }
 
-func TestReconcileWithTimeoutDeprecated(t *testing.T) {
```

Comment:
This is still deprecated (not removed) but handled via conversion, right?
So it's fine, we don't need a reconcile test for it, hopefully we have conversion tests :)

Reply:
yes these were taken at https://github.com/tektoncd/pipeline/blob/eb67e5a2d2df6255b77d34b3fe077722d6406ade/pkg/apis/pipeline/v1beta1/pipelinerun_conversion.go#L72 :) thanks for the checkin